pennylane/labs/resource_estimation/ops/op_math/symbolic.py

```diff
@@ -145,17 +559,172 @@ def controlled_resource_decomp(
 
     @staticmethod
     def tracking_name(base_class, base_params, num_ctrl_wires, num_ctrl_values, num_work_wires):
+        r"""Returns the tracking name built with the operator's parameters."""
         base_name = base_class.tracking_name(**base_params)
         return f"C({base_name},{num_ctrl_wires},{num_ctrl_values},{num_work_wires})"
 
 
 class ResourcePow(PowOperation, re.ResourceOperator):
-    """Resource class for the Pow symbolic operation."""
+    r"""Resource class for the symbolic PowOperation.
+
+    A symbolic class used to represent some base operation raised to a power.
+
+    Args:
+        base_class (Type[~.ResourceOperator]): The class type of the base operator to be raised to some power.
+        base_params (dict): the resource parameters required to extract the cost of the base operator
+        z (int): the power that the operator is being raised to
```

Comment:
The doc has different args:
![image](https://github.com/user-attachments/assets/9a54677a-ece2-4413-9368-974c4d6fced5)

Reply:
It is base_class, these are the arguments for the Op.resources() method, so they are named differently.